README.md

```diff
@@ -20,6 +21,86 @@ The following file types are known to be _writable_ with this library:
 - AnaMark multiple scales tuning (_.msf_)
 - HTML (for scale distribution and sharing)
 
+## Building the Library and Tests as Standalone Assets
+
+The AnaMark Tuning Library uses CMake to configure and build the solution.
+CMake is a widely available open source build configuraiton tool which can 
+create build assets for the development environment of your choice. You can
+install it from [cmake.org](https://cmake.org); if you install VS2017 or 2019
+you already have it; you can easily install it with homebrew on mac or apt on
+linux. 
+
+But once it is installed the recipe is simple. Run cmake to make your build
+assets, and either then open those build assets or run cmake to run a build.
+
+### Windows
+
+Open a Visual Studio command prompt. Check out this code and change to the
+directory. Type
+
+```
+cmake -Bbuild
+```
+
+and you will have in your build directory a visual studio solutions file which
+will allow you to create the library or test targets or you can build on the 
+command line with
+
+```
+cmake --build build --config Release
+```
+
+### macOS
+
+Run CMake to create xcode assets. For example
+
+```
+cmake -Bbuild -GXcode
+```
+
+You will then have an xcode file in the 'build' directory which you can
+open and use with xcode.  
+
+If you choose, you can build in xcode or from the command line with
+
+```
+cmake --build build --config Release 
+```
+
+Command line builds have far more readable output if you install
+and use the optional '[xcpretty](https://github.com/xcpretty/xcpretty)' utility.
+
+```
+cmake --build build --config Release | xcpretty
+```
+
+### Linux
+
+Very similar.
+
+```
+cmake -Bbuild
+cmake --build build --config Release
+```
+
+
+
+
+## Using the library in your code
+
+There are a variety of ways to use the library in your code. The two most common are
+
+1. Build the static library in a standalone fashion and set your include path to
```

Comment:
The steps for building above spews out many different files.  From my previous experience, I know the binary _anamark_tuning_test_ is an executable with a main function that can be ran to run the tests.  Is _libanamark_tuning_library_ the file I would need to include as a developer, not the binary?

Reply:
well it really depends on how you integrate. At this point in the project the best bet would be to just add the submodule and add the code to your build tree using your build mechanism. The library is not really set up right now to be super-transparently included in other code, which is why these instructions are hard to write. (Before this merge you would have just copied the files; with this merge you would also just copy the files, but with different directory names). So think of this merge as pre-saging the work we need to make inclusion easier (and there's basically three paths to that; make it easy to make static binary and link that, which I dislike because the folks have to get the binary library on all the platforms into their various build systems - that only works with really common ones like libsndfile I find; make the library header only and people copy it, ala catch2, at the end of a build phase; or make an amalgamated single C file which you can copy in, ala sqlite3).

Reply:
Alright good to know.  Thank you.  I'll keep this in mind as we move forward.